base/src/main/java/uk/ac/ebi/atlas/utils/ReactomeClient.java

```diff
@@ -32,7 +32,7 @@ public String fetchPathwayNameFailSafe(String reactomeId) {
             return StringUtils.trim(restTemplate.getForObject(url, String.class));
         } catch (RestClientException e) {
             LOGGER.warn("Reactome ID \"{}\" could not be found", reactomeId);
-            return null;
```

Comment:
This is called in a method `getLinkTextOrNull`, which is called in `Optional.ofNullable(...).orElse(<a decent default>)`. If you return "" are you not stopping the default from being displayed?